docker/Dockerfile

```diff
@@ -0,0 +1,18 @@
+# --- First stage
+FROM golang:1.14-buster AS build
+
+ARG REPOSITORY=Lirt
+ARG PLUGIN=velero-plugin-swift
+ENV GOPROXY=https://proxy.golang.org
+
+WORKDIR /go/src/github.com/${REPOSITORY}/${PLUGIN}
+COPY . .
+RUN CGO_ENABLED=0 go build -o /go/bin/${PLUGIN} .
+
+# --- Second stage
+FROM ubuntu:bionic
```

Comment:
why ubuntu? Would be better to use something more compact?

Reply:
It was just copy-paste from aws/gcp plugins. I'll try alpine.

Reply:
Rewritten to alpine. I also simpliefied entrypoint.